filebeat/input/syslog/input.go

```diff
@@ -0,0 +1,205 @@
+package syslog
+
+import (
+	"strings"
+	"sync"
+	"time"
+
+	"github.com/pkg/errors"
+
+	"github.com/elastic/beats/filebeat/channel"
+	"github.com/elastic/beats/filebeat/harvester"
+	"github.com/elastic/beats/filebeat/input"
+	"github.com/elastic/beats/filebeat/inputsource"
+	"github.com/elastic/beats/filebeat/util"
+	"github.com/elastic/beats/libbeat/beat"
+	"github.com/elastic/beats/libbeat/common"
+	"github.com/elastic/beats/libbeat/common/cfgwarn"
+	"github.com/elastic/beats/libbeat/logp"
+)
+
+// Parser is generated from a ragel state machine using the following command:
+//#go generate ragel -Z -G2 parser.rl -o parser.go
```

Comment:
You can write `//go:generate ragel -Z -G2 parser.rl -o parser.go` so that `go generate` will run the commands for you.

Reply:
Oh I've commented it for testing something, I will revert it back.